frontend/javascripts/oxalis/view/action-bar/download_modal_view.tsx

```diff
@@ -547,46 +555,42 @@ function _DownloadModalView({
                 }}
               >
                 Select the data you would like to download.
+                <Hint style={{ marginTop: 12 }}>
+                  An NML file will always be included with any download.
```

Comment:
will the NML always contain the skeletons? if not, then it's not possible to download volume + skeleton at the same time? and if yes, then it's not possible to download volume without skeletons? either way, it seems as this gives less control to the user? is this intended?

Reply:
Indeed, skeletons are always included. This was already the case before this PR. We might want to change that in the future, but that’s unrelated to this change.

Reply:
Ah, I see, then never mind :)